packages/axelar-wasm-std/src/counter.rs

```diff
@@ -23,7 +21,7 @@ impl<'a, T: AddAssign + Copy + Default + One + Serialize + DeserializeOwned> Cou
 
     pub fn incr(&self, store: &mut dyn Storage) -> StdResult<T> {
         let mut value = self.cur(store);
-        value += T::one();
+        value = value.saturating_add(&T::one());
```

Comment:
since `incr` already returns a result, I think it makes more sense to use a checked_add here, so it doesn't behave in an unexpected manner